docker-app/qfieldcloud/core/migrations/0082_project_attachments_file_storage.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.19 on 2025-05-06 14:42
+
+from django.db import migrations, models
+
+import qfieldcloud.core.models
+import qfieldcloud.core.validators
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0081_file_storage_project_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="project",
+            name="attachments_file_storage",
+            field=models.CharField(
+                default=qfieldcloud.core.models.get_project_file_storage_default,
+                help_text="Which file storage provider should be used for storing the project attachments files.",
+                max_length=100,
+                validators=[qfieldcloud.core.validators.file_storage_name_validator],
+                verbose_name="Attachments file storage",
+            ),
+        ),
+        migrations.RunSQL(
+            "UPDATE core_project SET attachments_file_storage = file_storage"
+        ),
```

Comment:
```suggestion
        migrations.RunSQL(
            "UPDATE core_project SET attachments_file_storage = file_storage",
            migrations.RunSQL.noop,
        ),
```

I kind like the explicit noop whenever possible, it gives a sense that things are finished.

I guess writing in pure python is not worth it, hope we don't hate ourselves about this in the future.